[2025-05-01 13:48:03,263: INFO: utils: NumExpr defaulting to 4 threads.]
[2025-05-01 14:15:36,836: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-05-01 14:15:36,851: INFO: common: yaml file: params.yaml loaded successfully]
[2025-05-01 14:15:36,856: INFO: common: created directory at: artifacts]
[2025-05-01 14:15:36,858: INFO: common: created directory at: artifacts\training]
[2025-05-01 14:15:37,367: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-05-01 14:16:40,491: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-05-01 14:16:40,494: INFO: common: yaml file: params.yaml loaded successfully]
[2025-05-01 14:16:40,496: INFO: common: created directory at: artifacts]
[2025-05-01 14:16:40,497: INFO: common: created directory at: artifacts\training]
[2025-05-01 14:16:40,709: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-05-01 15:02:31,826: INFO: utils: NumExpr defaulting to 4 threads.]
[2025-05-01 15:06:29,517: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-05-01 15:06:29,524: INFO: common: yaml file: params.yaml loaded successfully]
[2025-05-01 15:06:29,525: INFO: common: created directory at: artifacts]
[2025-05-01 15:06:29,525: INFO: common: created directory at: artifacts\training]
[2025-05-01 15:06:30,012: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-05-01 15:07:48,122: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-05-01 15:07:48,124: INFO: common: yaml file: params.yaml loaded successfully]
[2025-05-01 15:07:48,126: INFO: common: created directory at: artifacts]
[2025-05-01 15:07:48,128: INFO: common: created directory at: artifacts\training]
[2025-05-01 15:07:48,400: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-05-01 15:08:50,362: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-05-01 15:08:50,365: INFO: common: yaml file: params.yaml loaded successfully]
[2025-05-01 15:08:50,368: INFO: common: created directory at: artifacts]
[2025-05-01 15:08:50,369: INFO: common: created directory at: artifacts\training]
[2025-05-01 15:08:50,668: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-05-01 15:15:16,197: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-05-01 15:15:16,204: INFO: common: yaml file: params.yaml loaded successfully]
[2025-05-01 15:15:16,206: INFO: common: created directory at: artifacts]
[2025-05-01 15:15:16,210: INFO: common: created directory at: artifacts\training]
[2025-05-01 15:15:16,470: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-05-06 16:01:22,618: INFO: utils: NumExpr defaulting to 4 threads.]
[2025-05-06 16:01:28,623: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-05-06 16:01:28,623: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-05-06 16:01:28,623: INFO: common: yaml file: params.yaml loaded successfully]
[2025-05-06 16:01:28,636: INFO: common: created directory at: artifacts]
[2025-05-06 16:01:28,638: INFO: common: created directory at: artifacts/data_ingestion]
[2025-05-06 16:01:28,641: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-05-06 16:01:52,021: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-05-06 16:01:52,887: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-05-06 16:01:52,887: INFO: main: *******************]
[2025-05-06 16:01:52,887: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2025-05-06 16:01:52,898: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-05-06 16:01:52,898: INFO: common: yaml file: params.yaml loaded successfully]
[2025-05-06 16:01:52,900: INFO: common: created directory at: artifacts]
[2025-05-06 16:01:52,900: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-05-06 16:01:53,587: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-06 16:01:53,831: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-06 16:01:54,001: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2025-05-06 16:01:54,001: INFO: main: *******************]
[2025-05-06 16:01:54,001: INFO: main: >>>>>> stage Training started <<<<<<]
[2025-05-06 16:01:54,004: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-05-06 16:01:54,004: INFO: common: yaml file: params.yaml loaded successfully]
[2025-05-06 16:01:54,007: INFO: common: created directory at: artifacts]
[2025-05-06 16:01:54,007: INFO: common: created directory at: artifacts\training]
[2025-05-06 16:05:21,032: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-06 16:05:21,207: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2025-05-06 16:05:21,207: INFO: main: *******************]
[2025-05-06 16:05:21,209: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2025-05-06 16:05:21,215: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-05-06 16:05:21,222: INFO: common: yaml file: params.yaml loaded successfully]
[2025-05-06 16:05:21,222: INFO: common: created directory at: artifacts]
[2025-05-06 16:05:21,493: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-05-06 16:05:45,113: INFO: common: json file saved at: scores.json]
[2025-05-06 16:05:45,113: INFO: common: json file saved at: scores.json]
[2025-05-06 16:05:45,116: INFO: main: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2025-05-06 16:11:15,945: INFO: utils: NumExpr defaulting to 4 threads.]
[2025-05-06 16:11:21,025: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-05-06 16:11:21,031: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-05-06 16:11:21,033: INFO: common: yaml file: params.yaml loaded successfully]
[2025-05-06 16:11:21,035: INFO: common: created directory at: artifacts]
[2025-05-06 16:11:21,035: INFO: common: created directory at: artifacts/data_ingestion]
[2025-05-06 16:11:21,035: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-05-06 16:11:46,080: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-05-06 16:11:46,738: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-05-06 16:11:46,738: INFO: main: *******************]
[2025-05-06 16:11:46,738: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2025-05-06 16:11:46,738: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-05-06 16:11:46,738: INFO: common: yaml file: params.yaml loaded successfully]
[2025-05-06 16:11:46,738: INFO: common: created directory at: artifacts]
[2025-05-06 16:11:46,738: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-05-06 16:11:47,269: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-06 16:11:47,511: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-06 16:11:47,646: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2025-05-06 16:11:47,646: INFO: main: *******************]
[2025-05-06 16:11:47,646: INFO: main: >>>>>> stage Training started <<<<<<]
[2025-05-06 16:11:47,646: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-05-06 16:11:47,662: INFO: common: yaml file: params.yaml loaded successfully]
[2025-05-06 16:11:47,664: INFO: common: created directory at: artifacts]
[2025-05-06 16:11:47,664: INFO: common: created directory at: artifacts\training]
[2025-05-06 16:15:02,436: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-06 16:15:02,572: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2025-05-06 16:15:02,572: INFO: main: *******************]
[2025-05-06 16:15:02,572: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2025-05-06 16:15:02,572: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-05-06 16:15:02,572: INFO: common: yaml file: params.yaml loaded successfully]
[2025-05-06 16:15:02,583: INFO: common: created directory at: artifacts]
[2025-05-06 16:15:02,804: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-05-06 16:15:26,560: INFO: common: json file saved at: scores.json]
[2025-05-06 16:15:26,561: INFO: common: json file saved at: scores.json]
[2025-05-06 16:15:26,561: INFO: main: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2025-05-06 16:16:52,593: INFO: utils: NumExpr defaulting to 4 threads.]
[2025-05-06 16:16:58,406: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-05-06 16:16:58,408: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-05-06 16:16:58,408: INFO: common: yaml file: params.yaml loaded successfully]
[2025-05-06 16:16:58,422: INFO: common: created directory at: artifacts]
[2025-05-06 16:16:58,422: INFO: common: created directory at: artifacts/data_ingestion]
[2025-05-06 16:16:58,423: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-05-06 16:17:20,573: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-05-06 16:17:21,154: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-05-06 16:17:21,154: INFO: main: *******************]
[2025-05-06 16:17:21,154: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2025-05-06 16:17:21,157: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-05-06 16:17:21,159: INFO: common: yaml file: params.yaml loaded successfully]
[2025-05-06 16:17:21,159: INFO: common: created directory at: artifacts]
[2025-05-06 16:17:21,161: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-05-06 16:17:21,575: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-06 16:17:21,878: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-06 16:17:22,011: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2025-05-06 16:17:22,013: INFO: main: *******************]
[2025-05-06 16:17:22,013: INFO: main: >>>>>> stage Training started <<<<<<]
[2025-05-06 16:17:22,013: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-05-06 16:17:22,013: INFO: common: yaml file: params.yaml loaded successfully]
[2025-05-06 16:17:22,017: INFO: common: created directory at: artifacts]
[2025-05-06 16:17:22,019: INFO: common: created directory at: artifacts\training]
[2025-05-06 16:20:55,765: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-06 16:20:55,868: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2025-05-06 16:20:55,868: INFO: main: *******************]
[2025-05-06 16:20:55,868: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2025-05-06 16:20:55,874: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-05-06 16:20:55,884: INFO: common: yaml file: params.yaml loaded successfully]
[2025-05-06 16:20:55,887: INFO: common: created directory at: artifacts]
[2025-05-06 16:20:56,083: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-05-06 16:21:19,713: INFO: common: json file saved at: scores.json]
[2025-05-06 16:21:19,715: INFO: common: json file saved at: scores.json]
[2025-05-06 16:21:19,716: INFO: main: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2025-05-06 16:50:20,941: INFO: utils: NumExpr defaulting to 4 threads.]
[2025-05-06 16:50:27,937: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-05-06 16:50:27,940: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-05-06 16:50:27,943: INFO: common: yaml file: params.yaml loaded successfully]
[2025-05-06 16:50:27,945: INFO: common: created directory at: artifacts]
[2025-05-06 16:50:27,945: INFO: common: created directory at: artifacts/data_ingestion]
[2025-05-06 16:50:27,945: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-05-06 16:55:14,888: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-05-06 16:55:15,591: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-05-06 16:55:15,591: INFO: main: *******************]
[2025-05-06 16:55:15,591: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2025-05-06 16:55:15,593: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-05-06 16:55:15,594: INFO: common: yaml file: params.yaml loaded successfully]
[2025-05-06 16:55:15,595: INFO: common: created directory at: artifacts]
[2025-05-06 16:55:15,595: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-05-06 16:55:16,211: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-06 16:55:16,400: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-06 16:55:16,649: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2025-05-06 16:55:16,649: INFO: main: *******************]
[2025-05-06 16:55:16,649: INFO: main: >>>>>> stage Training started <<<<<<]
[2025-05-06 16:55:16,651: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-05-06 16:55:16,652: INFO: common: yaml file: params.yaml loaded successfully]
[2025-05-06 16:55:16,653: INFO: common: created directory at: artifacts]
[2025-05-06 16:55:16,655: INFO: common: created directory at: artifacts\training]
[2025-05-06 16:58:50,406: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-06 16:58:50,556: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2025-05-06 16:58:50,556: INFO: main: *******************]
[2025-05-06 16:58:50,556: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2025-05-06 16:58:50,556: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-05-06 16:58:50,556: INFO: common: yaml file: params.yaml loaded successfully]
[2025-05-06 16:58:50,573: INFO: common: created directory at: artifacts]
[2025-05-06 16:58:50,765: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-05-06 16:59:14,391: INFO: common: json file saved at: scores.json]
[2025-05-06 16:59:14,395: INFO: common: json file saved at: scores.json]
[2025-05-06 16:59:14,395: INFO: main: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2025-05-06 18:00:19,312: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2025-05-06 18:00:19,314: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-05-06 18:00:19,318: INFO: common: yaml file: params.yaml loaded successfully]
[2025-05-06 18:00:19,320: INFO: common: created directory at: artifacts]
[2025-05-06 18:00:19,320: INFO: common: created directory at: artifacts/data_ingestion]
[2025-05-06 18:00:19,321: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-05-06 18:00:41,614: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-05-06 18:00:43,158: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion Stage completed <<<<<<

x==========x]
[2025-05-06 18:00:59,437: INFO: utils: NumExpr defaulting to 4 threads.]
[2025-05-06 18:01:02,754: INFO: stage_02_prepare_base_model: *******************]
[2025-05-06 18:01:02,754: INFO: stage_02_prepare_base_model: >>>>>> stage Prepare base model started <<<<<<]
[2025-05-06 18:01:02,754: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-05-06 18:01:02,754: INFO: common: yaml file: params.yaml loaded successfully]
[2025-05-06 18:01:02,754: INFO: common: created directory at: artifacts]
[2025-05-06 18:01:02,754: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-05-06 18:01:03,283: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-06 18:01:03,550: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-06 18:01:03,701: INFO: stage_02_prepare_base_model: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2025-05-06 18:01:10,714: INFO: utils: NumExpr defaulting to 4 threads.]
[2025-05-06 18:01:13,757: INFO: stage_03_model_training: *******************]
[2025-05-06 18:01:13,757: INFO: stage_03_model_training: >>>>>> stage Training started <<<<<<]
[2025-05-06 18:01:13,759: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-05-06 18:01:13,761: INFO: common: yaml file: params.yaml loaded successfully]
[2025-05-06 18:01:13,762: INFO: common: created directory at: artifacts]
[2025-05-06 18:01:13,763: INFO: common: created directory at: artifacts\training]
[2025-05-06 18:04:52,460: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-06 18:04:52,612: INFO: stage_03_model_training: >>>>>> stage Training completed <<<<<<

x==========x]
[2025-05-06 18:05:02,462: INFO: utils: NumExpr defaulting to 4 threads.]
[2025-05-06 18:05:08,341: INFO: stage_04_model_evaluation: *******************]
[2025-05-06 18:05:08,341: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage started <<<<<<]
[2025-05-06 18:05:08,341: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-05-06 18:05:08,341: INFO: common: yaml file: params.yaml loaded successfully]
[2025-05-06 18:05:08,341: INFO: common: created directory at: artifacts]
[2025-05-06 18:05:08,608: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-05-06 18:05:29,383: INFO: common: json file saved at: scores.json]
[2025-05-06 18:05:29,385: INFO: common: json file saved at: scores.json]
[2025-05-06 18:05:29,385: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2025-05-06 18:21:48,838: INFO: utils: NumExpr defaulting to 4 threads.]
[2025-05-06 18:23:16,995: INFO: utils: NumExpr defaulting to 4 threads.]
[2025-05-06 18:23:19,018: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5001]
[2025-05-06 18:23:19,018: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-05-06 18:23:28,954: INFO: _internal: 127.0.0.1 - - [06/May/2025 18:23:28] "GET / HTTP/1.1" 200 -]
[2025-05-06 18:25:20,842: ERROR: app: Exception on /predict [POST]]
Traceback (most recent call last):
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\flask\app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\flask\app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\flask_cors\extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\flask\app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\flask\app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\flask_cors\decorator.py", line 121, in wrapped_function
    resp = make_response(f(*args, **kwargs))
                         ^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\Kidney-Disease-Classification-Deep-Learning-Project\app.py", line 43, in predictRoute
    result = clApp.classifier.predict()
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\Kidney-Disease-Classification-Deep-Learning-Project\src\cnnClassifier\pipeline\prediction.py", line 16, in predict
    model = load_model(os.path.join("model", "model.h5"))
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\keras\src\saving\saving_api.py", line 196, in load_model
    return legacy_h5_format.load_model_from_hdf5(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\keras\src\legacy\saving\legacy_h5_format.py", line 116, in load_model_from_hdf5
    f = h5py.File(filepath, mode="r")
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\h5py\_hl\files.py", line 562, in __init__
    fid = make_fid(name, mode, userblock_size, fapl, fcpl, swmr=swmr)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\h5py\_hl\files.py", line 235, in make_fid
    fid = h5f.open(name, flags, fapl=fapl)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "h5py\\_objects.pyx", line 54, in h5py._objects.with_phil.wrapper
  File "h5py\\_objects.pyx", line 55, in h5py._objects.with_phil.wrapper
  File "h5py\\h5f.pyx", line 102, in h5py.h5f.open
FileNotFoundError: [Errno 2] Unable to open file (unable to open file: name = 'model\model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
[2025-05-06 18:25:20,862: INFO: _internal: 127.0.0.1 - - [06/May/2025 18:25:20] "[35m[1mPOST /predict HTTP/1.1[0m" 500 -]
[2025-05-06 18:26:01,155: INFO: _internal: 127.0.0.1 - - [06/May/2025 18:26:01] "GET / HTTP/1.1" 200 -]
[2025-05-06 18:26:12,864: ERROR: app: Exception on /predict [POST]]
Traceback (most recent call last):
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\flask\app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\flask\app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\flask_cors\extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\flask\app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\flask\app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\flask_cors\decorator.py", line 121, in wrapped_function
    resp = make_response(f(*args, **kwargs))
                         ^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\Kidney-Disease-Classification-Deep-Learning-Project\app.py", line 43, in predictRoute
    result = clApp.classifier.predict()
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\Kidney-Disease-Classification-Deep-Learning-Project\src\cnnClassifier\pipeline\prediction.py", line 16, in predict
    model = load_model(os.path.join("model", "model.h5"))
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\keras\src\saving\saving_api.py", line 196, in load_model
    return legacy_h5_format.load_model_from_hdf5(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\keras\src\legacy\saving\legacy_h5_format.py", line 116, in load_model_from_hdf5
    f = h5py.File(filepath, mode="r")
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\h5py\_hl\files.py", line 562, in __init__
    fid = make_fid(name, mode, userblock_size, fapl, fcpl, swmr=swmr)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\h5py\_hl\files.py", line 235, in make_fid
    fid = h5f.open(name, flags, fapl=fapl)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "h5py\\_objects.pyx", line 54, in h5py._objects.with_phil.wrapper
  File "h5py\\_objects.pyx", line 55, in h5py._objects.with_phil.wrapper
  File "h5py\\h5f.pyx", line 102, in h5py.h5f.open
FileNotFoundError: [Errno 2] Unable to open file (unable to open file: name = 'model\model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
[2025-05-06 18:26:12,866: INFO: _internal: 127.0.0.1 - - [06/May/2025 18:26:12] "[35m[1mPOST /predict HTTP/1.1[0m" 500 -]
[2025-05-06 18:28:31,393: INFO: utils: NumExpr defaulting to 4 threads.]
[2025-05-06 18:28:38,215: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-05-06 18:28:38,215: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-05-06 18:28:38,229: INFO: common: yaml file: params.yaml loaded successfully]
[2025-05-06 18:28:38,233: INFO: common: created directory at: artifacts]
[2025-05-06 18:28:38,233: INFO: common: created directory at: artifacts/data_ingestion]
[2025-05-06 18:28:38,234: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-05-06 18:29:07,126: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-05-06 18:29:07,816: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-05-06 18:29:07,816: INFO: main: *******************]
[2025-05-06 18:29:07,816: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2025-05-06 18:29:07,820: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-05-06 18:29:07,820: INFO: common: yaml file: params.yaml loaded successfully]
[2025-05-06 18:29:07,820: INFO: common: created directory at: artifacts]
[2025-05-06 18:29:07,820: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-05-06 18:29:08,452: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-06 18:29:08,727: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-06 18:29:08,997: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2025-05-06 18:29:08,997: INFO: main: *******************]
[2025-05-06 18:29:08,997: INFO: main: >>>>>> stage Training started <<<<<<]
[2025-05-06 18:29:08,997: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-05-06 18:29:08,997: INFO: common: yaml file: params.yaml loaded successfully]
[2025-05-06 18:29:08,997: INFO: common: created directory at: artifacts]
[2025-05-06 18:29:08,997: INFO: common: created directory at: artifacts\training]
[2025-05-06 18:32:49,746: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-06 18:32:49,935: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2025-05-06 18:32:49,935: INFO: main: *******************]
[2025-05-06 18:32:49,935: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2025-05-06 18:32:49,938: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-05-06 18:32:49,944: INFO: common: yaml file: params.yaml loaded successfully]
[2025-05-06 18:32:49,947: INFO: common: created directory at: artifacts]
[2025-05-06 18:32:50,184: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-05-06 18:33:15,286: INFO: common: json file saved at: scores.json]
[2025-05-06 18:33:15,287: INFO: common: json file saved at: scores.json]
[2025-05-06 18:33:15,291: INFO: main: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2025-05-06 18:33:58,605: INFO: utils: NumExpr defaulting to 4 threads.]
[2025-05-06 18:34:01,408: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5001]
[2025-05-06 18:34:01,408: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-05-06 18:34:07,808: INFO: _internal: 127.0.0.1 - - [06/May/2025 18:34:07] "GET / HTTP/1.1" 200 -]
[2025-05-06 18:34:10,516: INFO: _internal: 127.0.0.1 - - [06/May/2025 18:34:10] "GET / HTTP/1.1" 200 -]
[2025-05-06 18:34:20,252: ERROR: app: Exception on /predict [POST]]
Traceback (most recent call last):
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\flask\app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\flask\app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\flask_cors\extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\flask\app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\flask\app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\flask_cors\decorator.py", line 121, in wrapped_function
    resp = make_response(f(*args, **kwargs))
                         ^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\Kidney-Disease-Classification-Deep-Learning-Project\app.py", line 43, in predictRoute
    result = clApp.classifier.predict()
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\Kidney-Disease-Classification-Deep-Learning-Project\src\cnnClassifier\pipeline\prediction.py", line 16, in predict
    model = load_model(os.path.join("model", "model.h5"))
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\keras\src\saving\saving_api.py", line 196, in load_model
    return legacy_h5_format.load_model_from_hdf5(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\keras\src\legacy\saving\legacy_h5_format.py", line 116, in load_model_from_hdf5
    f = h5py.File(filepath, mode="r")
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\h5py\_hl\files.py", line 562, in __init__
    fid = make_fid(name, mode, userblock_size, fapl, fcpl, swmr=swmr)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\h5py\_hl\files.py", line 235, in make_fid
    fid = h5f.open(name, flags, fapl=fapl)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "h5py\\_objects.pyx", line 54, in h5py._objects.with_phil.wrapper
  File "h5py\\_objects.pyx", line 55, in h5py._objects.with_phil.wrapper
  File "h5py\\h5f.pyx", line 102, in h5py.h5f.open
FileNotFoundError: [Errno 2] Unable to open file (unable to open file: name = 'model\model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
[2025-05-06 18:34:20,267: INFO: _internal: 127.0.0.1 - - [06/May/2025 18:34:20] "[35m[1mPOST /predict HTTP/1.1[0m" 500 -]
[2025-05-06 18:40:53,671: INFO: utils: NumExpr defaulting to 4 threads.]
[2025-05-06 18:40:55,553: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5001]
[2025-05-06 18:40:55,553: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-05-06 18:41:01,379: INFO: _internal: 127.0.0.1 - - [06/May/2025 18:41:01] "GET / HTTP/1.1" 200 -]
[2025-05-06 18:41:09,551: ERROR: app: Exception on /predict [POST]]
Traceback (most recent call last):
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\flask\app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\flask\app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\flask_cors\extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\flask\app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\flask\app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\flask_cors\decorator.py", line 121, in wrapped_function
    resp = make_response(f(*args, **kwargs))
                         ^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\Kidney-Disease-Classification-Deep-Learning-Project\app.py", line 43, in predictRoute
    result = clApp.classifier.predict()
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\Kidney-Disease-Classification-Deep-Learning-Project\src\cnnClassifier\pipeline\prediction.py", line 16, in predict
    model = load_model(os.path.join("model", "model.h5"))
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\keras\src\saving\saving_api.py", line 196, in load_model
    return legacy_h5_format.load_model_from_hdf5(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\keras\src\legacy\saving\legacy_h5_format.py", line 116, in load_model_from_hdf5
    f = h5py.File(filepath, mode="r")
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\h5py\_hl\files.py", line 562, in __init__
    fid = make_fid(name, mode, userblock_size, fapl, fcpl, swmr=swmr)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Harshitha\anaconda3\Lib\site-packages\h5py\_hl\files.py", line 235, in make_fid
    fid = h5f.open(name, flags, fapl=fapl)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "h5py\\_objects.pyx", line 54, in h5py._objects.with_phil.wrapper
  File "h5py\\_objects.pyx", line 55, in h5py._objects.with_phil.wrapper
  File "h5py\\h5f.pyx", line 102, in h5py.h5f.open
FileNotFoundError: [Errno 2] Unable to open file (unable to open file: name = 'model\model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
[2025-05-06 18:41:09,560: INFO: _internal: 127.0.0.1 - - [06/May/2025 18:41:09] "[35m[1mPOST /predict HTTP/1.1[0m" 500 -]
[2025-05-06 18:45:51,208: INFO: utils: NumExpr defaulting to 4 threads.]
[2025-05-06 18:45:54,214: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5001]
[2025-05-06 18:45:54,214: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-05-06 18:45:56,305: INFO: _internal: 127.0.0.1 - - [06/May/2025 18:45:56] "GET / HTTP/1.1" 200 -]
[2025-05-06 18:46:03,917: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-05-06 18:46:04,812: INFO: _internal: 127.0.0.1 - - [06/May/2025 18:46:04] "POST /predict HTTP/1.1" 200 -]
